### PR TITLE
upgraded antd v4 to v5

### PR DESCRIPTION
**Pull request by samouzegar10**
antd upgrade and adding scroll to all the tables
- css is messed up due to antd upgrade
- need to test the app sitewide

### CODE REVIEW

File: frontend/src/components/app/App.js [hunk 1]
```diff
@@ -241,19 +241,19 @@ const App = ({userID, usersByID, logOut, get}) => {
...
                 }
               </div>
             </div>
-            {isLoggedIn &&
+            {/* {isLoggedIn &&
               <div className='App__jumpBar'>
                 <JumpBar />
               </div>
-            }
+            } */}
```
Comment (nafiz1001) [inline frontend/src/components/app/App.js:256]: If it's not useful, it might as well be removed.

File: frontend/src/components/datasets/DatasetsListContent.tsx [hunk 2]
```diff
@@ -30,22 +31,23 @@ function DatasetsListContent() {
...
 
-	const columns = useFilteredColumns(
-						tableColumns, 
-						DATASET_FILTER_DEFINITIONS,
-						DATASET_FILTER_KEYS,
-						filters,
-						callbacks.setFilterCallback,
-						callbacks.setFilterOptionsCallback)
+  let tweakedColumns = useFilteredColumns(
+          tableColumns,
```
Comment (nafiz1001) [inline frontend/src/components/datasets/DatasetsListContent.tsx:35]: why did you rename from columns to tweakedColumns?

File: frontend/src/components/samples/SamplesListContent.tsx [hunk 3]
```diff
@@ -76,16 +76,22 @@ function SamplesListContent() {
...
+			[SampleColumnID.ID]: 130,
+			[SampleColumnID.KIND]: 130,
+			[SampleColumnID.COORDINATES]: 130,
+			[SampleColumnID.VOLUME]: 130,
+			[SampleColumnID.CONCENTRATION]: 130,
+			[SampleColumnID.CREATION_DATE]: 130,
+			[SampleColumnID.DEPLETED]: 130,
+      [SampleColumnID.NAME]: 130,
+      [SampleColumnID.PROJECT]: 130,
+      [SampleColumnID.INDIVIDUAL]: 130,
```
Comment (nafiz1001) [inline frontend/src/components/samples/SamplesListContent.tsx:91]: Indentation?

**Comment by nafiz1001** [comment]
v4
image

v5
image

**Comment by nafiz1001** [comment]
You might have forgotten to run:
```
npm i @ant-design/pro-layout
```

**Comment by nafiz1001** [comment]
Scrolling within table working great 👍🏼

**Comment by UlysseFG** [comment]
I think this is pretty nice overall. Some comments :
- Warning: [antd: Tabs] `Tabs.TabPane` is deprecated. Please use `items` instead. is appearing in the console ...It looks like it still works but it might not work soon.
- The tables scrolling is very nice but some table do not occupy the whole space available (white space left bellow the page).
- Most of the column are large enough but name wrapping make reading the sample name very hard. I would allocate more space to sample names.
- Available action and Prefill template drop downs are aligned to the center. They should be aligned left.

These are the elements that jumped to my face. The Overall change is very positive. The tables feel more natural and the rounded design is quite pleasing to me.

File: frontend/src/components/app/App.js (hunk 1 above)
Comment (samouzegar10) [reply frontend/src/components/app/App.js:256]: i dont think this is the right assessment over this component

Comment (samouzegar10) [reply frontend/src/components/app/App.js:256]: the component needs to be configured correctly if you guys want to use it. 
it would help the user jump from one page to to another, is this something freezeman would need?

File: frontend/src/components/datasets/DatasetsListContent.tsx (hunk 2 above)
Comment (samouzegar10) [reply frontend/src/components/datasets/DatasetsListContent.tsx:35]: i tried mirroring another component file as reference. it didn't quite work and i was able to reach a simpler way

File: frontend/src/components/app/App.js (hunk 1 above)
Comment (UlysseFG) [reply frontend/src/components/app/App.js:256]: It was requested multiple times by David (id think he likes it) i never use it myself but i don't see why we would remove a working component ... The upgrade breaks it ? Maybe the css can be replaced with some component style ? I really would prefer we don't start removing stuff just because we upgrade antd ...

File: frontend/src/components/datasets/DatasetsListContent.tsx (hunk 2 above)
Comment (UlysseFG) [reply frontend/src/components/datasets/DatasetsListContent.tsx:35]: I think Nafiz mention this because the change of variable name to "tweakedColumns" do not really carry more information than the simpler "columns" that was already there.

**Comment by samouzegar10** [comment]
> /pro-layout

not at all, its installed when i do npm install, isnt it in your node_modules?

File: frontend/src/components/app/App.js (hunk 1 above)
Comment (UlysseFG) [reply frontend/src/components/app/App.js:256]: As far as i can see as long as it is not commented out it works fine ...

**Comment by UlysseFG** [comment]
I would suggest that we get this merged quickly (once we have addressed the current comments) in order to have Andres use the new version as he tests other features. He may unearth hidden issues that way.

**Comment by UlysseFG** [comment]
> http://localhost:9000/libraries/list
> 
> v4 image
> 
> v5 image

Problem is likely here "columns = setDynamicSorters(LIBRARY_TABLE_COLUMNS, [" in the library list component ... You should use columns instead of LIBRARY_TABLE_COLUMNS...

**Comment by nafiz1001** [comment]
> > /pro-layout
> 
> not at all, its installed when i do npm install, isnt it in your node_modules?

It's missing in the package.json. I had to manually intervene with `npm install`.

**Comment by nafiz1001** [comment]
> > > /pro-layout
> > 
> > 
> > not at all, its installed when i do npm install, isnt it in your node_modules?
> 
> It's missing in the `package.json`. I had to manually intervene with `npm i @ant-design/pro-layout`.

I tested your assumption. Never mind. It started working again 🤷🏼

File: frontend/src/components/app/App.js (hunk 1 above)
Comment (samouzegar10) [reply frontend/src/components/app/App.js:256]: the upgrade did make this component look out of place when i first started to work on this task, i decided to remove it for the meantime while i worked on the other more breaking parts of the css. i've tried to use it but i couldn't get anywhere with it. i keep receiving 5XX server error codes. 

I propose to open a ticket for it on redmine and investigate it

Comment (samouzegar10) [reply frontend/src/components/app/App.js]: disregard last line of previous comment, no ticket needed

File: frontend/src/components/datasets/ExpandableTableDatasetComments.tsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ import { Dataset } from "../../models/frontend_models";
 export default function ExpandableTableDatasetComments(): NonNullable<TableProps<any>['expandable']> {
   return (
     {
-      columnTitle: <div>Comments</div>,
+      columnTitle: <CommentOutlined />,
```
Comment (UlysseFG) [inline frontend/src/components/datasets/ExpandableTableDatasetComments.tsx:12]: I personally prefer to keep text in the headers ... those icons are really not that helpful when it comes to understanding their meaning ...

**Comment by UlysseFG** [comment]
Make sure to rebase so we can test all pages looks. At the moment there is a few pages without anything on them. It is likely the issue we fixed in master, but we can't be sure unless you rebase ...

**Comment by UlysseFG** [comment]
The tables seem to have more horizontal scrolling than the current version. Maybe we can try to fit the tables that currently fit on a single screen width so they still do. The others we might actually give a bit more space to fields like Name.

File: frontend/src/components/datasets/ExpandableTableDatasetComments.tsx (hunk 4 above)
Comment (samouzegar10) [reply frontend/src/components/datasets/ExpandableTableDatasetComments.tsx:12]: i have no quarrels with either or, although i would be interested to know how the icon approach will be received by the users. i can set it back to the string 'comments' if you are ok with it. 

(nowadays, thanks to fact that almost everyone has a smart phone with them and the main means of communicating on the some of these modern apps is to display icons)

**Comment by samouzegar10** [comment]
> The tables seem to have more horizontal scrolling than the current version. Maybe we can try to fit the tables that currently fit on a single screen width so they still do. The others we might actually give a bit more space to fields like Name.

i do not disagree, we can definitely make the tables much better, although, this one comment would require me to spend much more time on the pr. we can open an issue on redmine for this big task you have mentioned in this comment

**Comment by nafiz1001** [comment]
I recommend doing `git reset --hard a00bd16` or `git reset --soft a00bd16` depending on your preference. That should bring you back to [indent issues](https://github.com/c3g/freezeman/pull/861/commits/a00bd16e508efef2c30684f39babf16d0bcafba6) before you made those merges.

**Comment by samouzegar10** [comment]
started a new  branch with that commit
